tests/units/common/parsing/test_parser.py

```diff
@@ -20,13 +20,142 @@
     </Root>
 """
 
+publisher_parsed_article = {
```

Comment:
@Neodia I would move these to a separate file (fixture) and use https://pypi.org/project/pytest-datadir/ to read it (we already have it)

Reply:
Ernesta also gave the idea of `pytest-datadir`. I said it was a bad idea because it would copy data as I'm using the same that we are pushing to the `docker-compose` FTP server. What do you think ?

Reply:
Just saw your other comment. Will change this

Reply:
Done

Reply:
@Neodia why not moving these two in a separate `json` files?

Reply:
Sorry I mingled this comment with the other with the datadir. Done 👍